app/assets/javascripts/classes/reports/EditReportViewModel.ts

```diff
@@ -77,7 +77,7 @@ class EditReportViewModel {
                 this.recipients.push(model);
             });
         }).fail(() => {
-            this.alertMessage("Report not found");
+            this.alertMessages(["Report not found"]);
```

Comment:
Ideally on a 404 for the report we would return the error information from the server instead of hard coding the language specific text in the client.